packages/cosmos_ui_components/lib/components/cosmos_bottom_sheet_header.dart

```diff
@@ -23,6 +23,7 @@ class CosmosBottomSheetHeader extends StatelessWidget {
     return Column(
       mainAxisSize: MainAxisSize.min,
       children: [
+        SizedBox(height: CosmosTheme.of(context).spacingL),
```

Comment:
while this adds spacing to the header for situations with no `leading` nor `actions`, it breaks the designs for those with leading/actions:
<img width="319" alt="image" src="https://user-images.githubusercontent.com/1711474/143866854-9acc929a-e807-450a-a0ff-fc5d9846c54d.png">
<img width="324" alt="CleanShot 2021-11-29 at 13 19 56@2x" src="https://user-images.githubusercontent.com/1711474/143867119-852c1514-7b79-4333-8ee6-814494882b6c.png">
Also, the spacing is too small according to designs:
<img width="495" alt="CleanShot 2021-11-29 at 13 21 05@2x" src="https://user-images.githubusercontent.com/1711474/143867290-084e20b1-e8a3-4b68-a769-728dba2d500f.png">

Reply:
It doesn't break the design for those with leading/actions. I will remove that extra spacing from the template in another short PR. Also, 40 isn't something which we have in spacings. It is either 16 or 32. Should I create one?

Reply:
ah, yes, it looks perfect on the sheet with leading/actions. Could you create the `spacingXXL` for 40 px then? :)

Reply:
Thanks, updated :)